Mlem/ContentView.swift

```diff
@@ -30,13 +30,25 @@ struct ContentView: View {
     @State private var tabSelection: TabSelection = .feeds
     @State private var tabNavigation: any FancyTabBarSelection = TabSelection._tabBarNavigation
     @GestureState private var isDetectingLongPress = false
+    @State private var isAppLockAuthenticated: Bool = true
     
     @State private var isPresentingAccountSwitcher: Bool = false
     @State private var tokenRefreshAccount: SavedAccount?
     
     @AppStorage("showInboxUnreadBadge") var showInboxUnreadBadge: Bool = true
     @AppStorage("homeButtonExists") var homeButtonExists: Bool = false
     @AppStorage("allowTabBarSwipeUpGesture") var allowTabBarSwipeUpGesture: Bool = true
+    @AppStorage("appLock") var appLock: AppLock = .disabled
+
+    @StateObject var biometricUnlock = BiometricUnlock()
+    
+    var isAppLocked: Bool {
+        if appLock != .disabled, !biometricUnlock.isUnlocked {
+            return true
+        }
+        
+        return false
+    }
```

Comment:
This can be simplified a little

```suggestion
    var isAppLocked: Bool {
        appLock != .disabled && !biometricUnlock.isUnlocked
    }
```